api/src/main/java/com/datastrato/gravitino/annotation/Stable.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * APIs that can evolve while retaining compatibility for feature releases (0.5.0 to 0.6.0). The
```

Comment:
Maybe we can add the version info as an attribute to the annotation? like:

```java
  /**
    * Optional: Version when the API was marked as Stable.
    */
    String since() default "";
```

Reply:
Sure, let me check.